PowerPointLabs/PowerPointLabs/ShortcutsLab/ConvertToPicture.cs

```diff
@@ -69,6 +71,15 @@ private static void ConvertToPictureForShape(PowerPoint.Shape shape)
             pic.Left = x + (width - pic.Width) / 2;
             pic.Top = y + (height - pic.Height) / 2;
             pic.Rotation = rotation;
+            // move picture to original z-order
```

Comment:
Could rename zOrder to originalZOrder to be clearer.

Reply:
@initialshl Code passed all tests after latest changes